src/commissaire_http/server/cli.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# Copyright (C) 2016  Red Hat, Inc
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Commissaire HTTP based application server.
+"""
+import argparse
+import logging
+
+from commissaire_http.server.routes import DISPATCHER
+from commissaire_http import CommissaireHttpServer, parse_args
+
+
+# TODO: Make this configurable
+for name in ('Dispatcher', 'Router', 'Bus', 'CommissaireHttpServer'):
```

Comment:
Seems like we could accommodate logger specifications in the JSON config pretty easily.